libpod/container_internal.go

```diff
@@ -183,6 +183,7 @@ func newContainer(rspec *spec.Spec, lockDir string) (*Container, error) {
 	ctr := new(Container)
 	ctr.config = new(ContainerConfig)
 	ctr.state = new(containerState)
+	ctr.state.ExitCode = -1
```

Comment:
Why do we initialize exit code to -1?

Reply:
Right now we are exiting with 0, when we don't know if their was success. Perhaps we should exit with 125, now that I think about it. since that indicates that something went wrong with the command, rather then the container.

Reply:
I don't know if setting this to -1 here is the right approach. Maybe we need an `Exited` bool to check if the container has actually died.